plan/physical_plans.go

```diff
@@ -72,7 +72,8 @@ type PhysicalTableReader struct {
 	*basePlan
 	basePhysicalPlan
 
-	TablePlan PhysicalPlan
+	TablePlans []PhysicalPlan
```

Comment:
Add comment or make the name more meaningful, because it is not obvious that `TablePlans` is flattened `tablePlan` in this context.